writeengine/wrapper/writeengine.h

```diff
@@ -72,15 +72,23 @@ class Log;
 // for truly long running transactions.
 struct TxnLBIDRec
 {
-    std::tr1::unordered_map<BRM::LBID_t, uint32_t> m_LBIDMap;
-    uint32_t        m_lastSeqnum;
+    std::tr1::unordered_set<BRM::LBID_t> m_LBIDSet;
+    std::vector<BRM::LBID_t> m_LBIDs;
+    std::vector<execplan::CalpontSystemCatalog::ColDataType> m_ColTypes;
```

Comment:
`m_colDataTypes` will be more appropriate terminology here, since colType has a different meaning in ColumnStore. See dbcon/execplan/calpontsystemcatalog.h: `struct ColType` has colDataType as an attribute.